posts/TEMPLATE/index.md

```diff
@@ -1,9 +1,73 @@
 ---
 date: 2030-12-31 # Auto updates on publish.
-title: Hello World!
+title: Get Pull Request Approval with the GitHub API (v4)
 author: sascha.lampalzer@cybertec.at
-tags: ["list-tags-here", "🙂"] # max. 10 tags; lowercase; dash-separated
-description: "This is an example" # max. 300 chars.
+tags: ["github", "graphql", "typescript"] # max. 10 tags; lowercase; dash-separated
+description: "Natively the GitHub API does not provide a way to get a pull request's approval status. Here's a workaround." # max. 300 chars.
 ---
 
-This is just an example post, don't worry about it. We use this internally as a template for awesome, future posts.
+Natively the GitHub API does not provide a way to get a pull request's approval status. Here's a workaround.
+
+It is necessary to compare the date of the newest commit and the date of last approval, because new commits automatically invalidate any approvals (Default behavior, can be configured.)
```

Comment:
```suggestion
It is necessary to compare the date of the newest commit and the date of last approval, because new commits automatically invalidate any approvals (default behavior, can be configured).
```